docs/architecture/TECHNICAL_DATA_MODEL.md

```
@@ -425,6 +425,82 @@ File-system based permissions:
+```json
+{
+  "locked_by": "user123",
+  "locked_at": 1704110400,
+  "expires_at": 1704114000,
+  "operation": "edit",
+  "pid": 12345
+}
+```
+
+Lock file naming convention:
+- `{item-id}.lock`
```

markdownlint-cli2 (0.17.2)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)